This is a testharness.js-based test.
PASS respondWith on script created FetchEvent should not crash 
PASS Calling respondWith should not crash 
Harness: the test ran to completion.

